var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -93,6 +94,10 @@ def build(self, spec, prefix):
                                                  loadflags=link_flag))
         os.chmod(_BUILD_NEURODAMUS_FNAME, 0o770)
 
+    def get_git_fetcher(self):
```

Comment:
Actually I'd consider moving the two lines of this function into line 137, or factor them out.